### PR TITLE
Swap bucket-assets for @artsy/bucket-assets

### PR DESCRIPTION
**Pull request by eessex**
Updates to @artsy-namespaced `bucket-assets` lib, which has been updated (caused deprecation conflicts when updating `superagent` due to `knox` and `mime` versions).

### CODE REVIEW

File: scripts/publish-assets.sh [hunk 1]
```diff
@@ -10,4 +10,4 @@ fi
 
 export COMMIT_HASH=`cat COMMIT_HASH.txt`
 
-yarn assets && yarn bucket-assets -b $S3_BUCKET
+yarn assets && ./node_modules/.bin/bucket-assets -b $S3_BUCKET
```
Comment (damassi) [inline scripts/publish-assets.sh]: Why is this change necessary?

Comment (eessex) [reply scripts/publish-assets.sh]: I was worried that the name change might affect this command, but might not be necessary -- will remove and see if it works as expected on staging deploy.

Comment (damassi) [reply scripts/publish-assets.sh]: Nope, the `@foo` is a npm namespace scope: https://docs.npmjs.com/about-scopes for grouping packages. 
 What matters is the `node_modules/.bin/file-name` reference as those are what's made available to npm or yarn via `yarn x` commands.
